client/reader/full-post/index.jsx

```diff
@@ -53,6 +53,7 @@ var CommentButton = require( 'components/comment-button' ),
 
 import PostExcerpt from 'components/post-excerpt';
 import { getPostTotalCommentsCount } from 'state/comments/selectors';
+import GalleryView from 'components/tinymce/plugins/wpcom-view/gallery-view';
```

Comment:
```GalleryView``` is heavily tied into usage of Shortcodes used by the the ```tinymce``` plugin. If the eventual goal is to create a new Slideshow, it may not make sense to move this to ```components/slideshow/gallery-view```. Although, I agree its not ideal.